aquarius/events/processors.py

```diff
@@ -32,7 +32,14 @@
 
 class EventProcessor(ABC):
     def __init__(
-        self, event, oceandb, web3, ecies_account, allowed_publishers, purgatory
+        self,
+        event,
+        oceandb,
+        web3,
+        ecies_account,
+        allowed_publishers,
+        purgatory,
+        chain_id,
```

Comment:
Because self._chain_id=web3.eth.chain_id has 2 major drawbacks:
 - it's slow  (at least 500 msec)
 - it's an extra RPC call that counts to your infura daily limit
 And chain_id it's not changing